packagers/pnpm-workspaces-custom-output/workspace/sub-project-1/script.ts

```diff
@@ -0,0 +1,25 @@
+import { PrismaClient } from '.prisma/client'
```

Comment:
This is equivalent to importing from @prisma/client, isn't it?

Reply:
Not in a symlinked/hoisted/workspace. Because `@prisma/client` is in it's own folder, it will find the default generated client which throws an error message. Those generated clients are found in the sub-projects `node_modules`. However, the other test `pnpm-workspaces-default-output` will let two clients overwrite each other.